bc_obps/registration/api/user/user_app_role.py

```diff
@@ -1,15 +1,13 @@
-from registration.models import User
+from registration.decorators import handle_http_errors
+from service.data_access_service.user_service import UserDataAccessService
 from registration.schema import UserAppRoleOut, Message
 from registration.api.api_base import router
 from ninja.responses import codes_4xx
 
-# endpoint to return user's role_name if user exists in user table
+
 @router.get(
     "user/user-app-role/{user_guid}", response={200: UserAppRoleOut, codes_4xx: Message}, url_name="get_user_role"
 )
+@handle_http_errors()
 def get_user_role(request, user_guid: str):
-    try:
-        user = User.objects.only('app_role').select_related('app_role').get(user_guid=user_guid)
-    except User.DoesNotExist:
-        return 404, {"message": "No matching user found"}
-    return 200, user.app_role
+    return 200, UserDataAccessService.get_app_role(user_guid)
```

Comment:
@Sepehr-Sobhani , I think returning a 200 is optional because django ninja will do it on success. Would you recommend including it anyway? Sometimes I did and sometimes I didn't in the refactor tickets, so I can standardize to whatever you think is best

Reply:
Correct, Django Ninja does default to a 200 status code on successful requests. However, it's still beneficial to explicitly set it to 200 for consistency and clarity, especially when different status codes are used within the same function. Standardizing this approach can help prevent confusion among developers who are new to the codebase.